functional/FunctionalTest.java

```diff
@@ -437,12 +437,12 @@ public static void putObject_test2() throws Exception {
     long startTime = System.currentTimeMillis();
     try {
       String filename = createFile6Mb();
-      client.putObject(bucketName, filename, filename);
+      client.putObject(bucketName, filename, filename, null, null,(ServerSideEncryption)null , null);
```

Comment:
is it required to type cast `null` here?

Reply:
removed